cli/src/main.rs

```diff
@@ -143,6 +144,46 @@ fn build_app<'a, 'b>() -> App<'a, 'b> {
                     )
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("find-path")
+                 .about("Fetch possible payment paths")
+                            .arg(
+                                Arg::with_name("destination_account")
+                                    .long("destination_account")
+                                    .takes_value(true)
+                                    .help("The destination account id of the payment path to query"),
+                            )
+                            .arg(
+                                Arg::with_name("source_account")
+                                    .long("source_account")
+                                    .takes_value(true)
+                                    .help("The source account id of the payment path to query"),
+                            )
+                            .arg(
+                                Arg::with_name("destination_asset_type")
+                                    .long("destination_asset_type")
+                                    .takes_value(true)
+                                    .help("The type of the destination asset"),
+                            )
+                            .arg(
+                                Arg::with_name("destination_asset_issuer")
+                                    .long("destination_asset_issuer")
+                                    .takes_value(true)
+                                    .help("The asset issuer account id for the destination asset"),
+                            )
+                            .arg(
+                                Arg::with_name("destination_asset_code")
+                                    .long("destination_asset_code")
+                                    .takes_value(true)
+                                    .help("The code of the destination asset"),
+                            )
+                            .arg(
```

Comment:
We've added a new asset_identifier struct to the CLI https://github.com/kbacha/stellar-sdk/blob/master/cli/src/asset_identifier.rs and we can now shorten the asset inputs like so https://github.com/kbacha/stellar-sdk/blob/master/cli/src/main.rs#L175-L180

Reply:
Ok sweet, noted. Got so wrapped up in work and that memo PR from last week I missed some of these :fire: changes.